### ISSUE TITLE
Edit scores page maybe?

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2968]
Is this the right description for 1-5?
[Uploading... Screen Shot 2021-05-27 at 11.03.36 AM.png]

**Comment by nickiredescent** [comment]
This looks good to me

**Comment by ophelieh** [comment]
Looks good to me too! One side note - a few people emailed us last season to ask why they couldn't see the feedback. Could we make it more visible that each portion is clickable? 

![Screen Shot 2021-06-07 at 11 39 49 AM (2)](https://user-images.githubusercontent.com/32206421/121057796-46f57000-c785-11eb-8265-8d0535deb4b1.png)

**Comment by rsgonzal** [comment]
that is good feedback, would a sentence work that says "click for feedback" or @viviancan any ideas on some visual clues? a down arrow?

**Comment by ophelieh** [comment]
Ohh I like that text or even an arrow or a "+" could be helpful :)

**Comment by rsgonzal** [comment]
Ok! I think the top left looks best, thoughts?

![Screen Shot 2021-06-10 at 2.52.24 PM.png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/f0cbee35-4c41-4cdd-94c8-21a3f6ebcdb0)

**Comment by ophelieh** [comment]
I like both the top and bottom left :) But for an international audience, maybe the top left makes the most sense.

**Comment by Judith-Ahumada** [comment]
I agree with you both about the top left. Do you guys think including the text "Click for feedback" somewhere at the top in the maroon box will look to cluttered?

**Comment by mdancho** [comment]
Top left looks most clean and easy to read from my perspective. Adding feedback or click for feedback could be helpful also.

**Comment by viviancan** [comment]
PR #2971

**Comment by viviancan** [comment]
#2973

**Comment by viviancan** [comment]
Update 6/15/21 - Top left mockup on QA now!

**Comment by viviancan** [comment]
![Screen Shot 2021-06-15 at 9.44.48 AM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/514805ac-510b-420b-acc4-8c2b1943ebf8)

### QA Steps

1. Login as a student that has submitted a project
2. Select the view your scores and certificate button
3. Confirm there is a little maroon chevron icon and that you can expand the sections

**Comment by ophelieh** [comment]
Looks great! :)

**Comment by rsgonzal** [comment]
woo! working/looking as expected.
